sdk/botservice/azure-mgmt-botservice/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Release History
 
+## 0.2.0 (2019-05-21)
+
```

Comment:
@changlong-liu can you take this opportunity to compute the changelog and put it here? It seems it was forgotten when 0.2.0 was released

Reply:
Yes @lmazuel .  
Since this PR has already merged, I create a new PR https://github.com/Azure/azure-sdk-for-python/pull/12434 to add the 0.2.0 changelog. 

Hi @chidozieononiwu , would you help to review it? Thank you:)